.circleci/continue_config.yml

```diff
@@ -218,16 +220,16 @@ jobs:
       - checkout
       - run:
           name: install dependencies
-          command: npm install && npx lerna bootstrap
+          command: npm install
```

Comment:
Asking because idk, so now there's no need for `npx lerna bootstrap ` but we do `npm install`. Does the package linking handled in that case?

Also we need to update the docs in this case

Reply:
yes it handles everything by itself. I needed to add the `workspace` key in the main `package.json`.